summoner-cli/src/Summoner/Project.hs

```diff
@@ -155,25 +157,61 @@ generateProject settingsNoUpload isOffline projectName Config{..} = do
         Last prelude@(Just (CustomPrelude p _)) ->
             prelude <$ successMessage ("Custom prelude " <> p <> " will be used in the project")
 
+    getNixPkgSet :: Bool -> IO (Maybe NixPkgSet)
+    getNixPkgSet usingNix = if usingNix
+        then case cNixPkgSet of
+            Last Nothing -> do
+                let yesDo, noDo :: IO (Maybe NixPkgSet)
+                    yesDo = do
+                        o <- queryNotNull "Nix package set GitHub owner name: "
+                        r <- queryNotNull "Nix package set GitHub repo name: "
+                        rv <- queryNotNull "Nix package set Git revision: "
+                        s  <- queryNotNull "Nix package set unpacked SHA256: "
+                        let pkgSet = NixPkgSet { npsOwner = o, npsRepo = r, npsRev = rv, npsSha = s }
+                        successMessage ("The nix package set " <> showNixPkgSet pkgSet <> " will be used in the project")
+                        pure $ Just pkgSet
+                    noDo = do
+                        successMessage ("The default nix package set " <> showNixPkgSet defaultNixPkgSet <> " will be used in the project")
+                        pure $ Just defaultNixPkgSet
+                chooseYesNo (mkDefaultYesNoPrompt "nix package set") yesDo noDo
```

Comment:
Probably, add a more clear comment about what Yes and No will mean for the user in this case (that the default will be used if the user says no if I understand this correctly).